test/lib/merge-reports/data-tree.js

```diff
@@ -14,9 +13,7 @@ describe('lib/merge-reports/data-tree', () => {
         return DataTree.create(initialData, destPath);
     };
 
-    const mkData_ = (data = {}) => {
-        return _.defaults(data, {suites: [], skips: []});
-    };
+    const mkData_ = (data = {}) => data;
```

Comment:
Do you leave `mkData_`-function for the future modifications/extensions? Now it looks like unnecessary function.

Reply:
fixed